components/base/kinematicbase/kinematics.go

```diff
@@ -202,3 +222,42 @@ func CollisionGeometry(cfg *referenceframe.LinkConfig) ([]spatialmath.Geometry,
 	}
 	return []spatialmath.Geometry{sphere}, nil
 }
+
+// newValidRegionCapsule returns a capsule which defines the valid regions for a base to be when moving to a waypoint.
+// The valid region is all points that are deviationThreshold (mm) distance away from the line segment between the
+// starting and ending waypoints. This capsule is used to detect whether a base leaves this region and has thus deviated
+// too far from its path.
+func newValidRegionCapsule(starting, desired []referenceframe.Input, deviationThreshold float64) (spatialmath.Geometry, error) {
```

Comment:
We don't need the deviation threshold as an argument since its always the same thing being passed in.  Can just use the constant directly in the function